gpflow/quadrature/deprecated.py

```diff
@@ -136,7 +136,7 @@ def ndiagquad(funcs, H: int, Fmu, Fvar, logspace: bool = False, **Ys):
 
     def wrapper(old_fun):
         def new_fun(X, **Ys):
-            Xs = tf.unstack(X, axis=-1)
+            Xs = tf.unstack(tf.expand_dims(X, axis=-2), axis=-1)
```

Comment:
why this change ?

Reply:
This was necessary to make the wrapper work with the new shape.